src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Win32.cs

```diff
@@ -345,10 +337,8 @@ private static int GetNewBufferSize(int existingBufferSize, int requiredSize)
 
             if (requiredSize == 0)
             {
-                //
                 // On some old OS like win2000, requiredSize will not be set if the buffer
```

Comment:
We do not support these very old OSes. Delete this workaround for old OS bug?

Reply:
I've removed the win2000 workaround and added `Debug.Assert` to ensure that the value returned by the OS is valid